ipywidgets/package.json

```diff
@@ -1,56 +1,23 @@
 {
-  "name": "jupyter-js-widgets",
-  "version": "4.1.0",
-  "description": "IPython widget Javascript",
-  "author": "IPython",
-  "license": "BSD-3-Clause",
+  "name": "ipywidgets",
+  "version": "1.0.0",
+  "description": "ipywidgets JavaScript dependencies",
   "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "postinstall": "ncp ../jupyter-js-widgets ./node_modules/jupyter-js-widgets"
+  },
   "repository": {
     "type": "git",
-    "url": "https://github.com/ipython/ipywidgets.git"
+    "url": "git+https://github.com/ipython/ipywidgets.git"
   },
-  "scripts": {
-    "build": "npm run css",
-    "bower": "bower install --config.interactive=false",
-    "postinstall": "npm run bower",
-    "precss": "rimraf ./static/widgets/css/",
-    "css": "node build_css.js",
-    "test": "npm run test:firefox && npm run test:chrome",
-    "test:firefox": "karma start karma.conf.js --browsers Firefox --log-level debug",
-    "test:chrome": "karma start karma.conf.js --log-level debug",
-    "prepublish": "npm run bower && npm run build",
-    "publish": "npm run publish:pypi",
-    "publish:pypi": "cd .. && python setup.py sdist upload && python setup.py bdist_wheel upload && cd ipywidgets",
-    "version": "node copy_version.js && git add _version.py",
-    "postversion": "node copy_version.js --dev && git add _version.py && git commit -m \"Back to dev\""
-  },
-  "devDependencies": {
-    "babel-core": "^6.4.0",
-    "babel-loader": "^6.2.1",
-    "babel-preset-es2015": "^6.3.13",
-    "bower": "^1.5.3",
-    "chai": "^3.4.1",
-    "clean-css": "^3.4.6",
-    "karma": "^0.13.15",
-    "karma-chai": "^0.1.0",
-    "karma-chrome-launcher": "^0.2.2",
-    "karma-firefox-launcher": "^0.1.7",
-    "karma-mocha": "^0.2.1",
-    "karma-mocha-reporter": "^1.1.3",
-    "karma-sinon-chai": "^1.1.0",
-    "karma-webpack": "^1.7.0",
-    "less": "^2.5.3",
-    "mocha": "^2.3.4",
-    "rimraf": "^2.4.1",
-    "sinon": "^1.17.2",
-    "spawn-sync": "^1.0.14",
-    "webpack": "^1.12.11"
+  "author": "Jupyter",
```

Comment:
In other jupyter-js-*, it is "Project Jupyter". Besides, rather that calling this "ipywidgets JavaScript dependencies", it could be "Jupyter Interactive Widgets".